udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -37,484 +38,559 @@
 {% endcache %}
 {% endblock %}
 
+{% block toolbar %}
+{% cache cache_duration, 'dataset-toolbar', dataset.id|string, g.lang_code, dataset.last_modified|string %}
+{{ follow_btn(dataset) }}
+{{ hook('dataset.display.explore-button') }}
+{% if can_edit %}
+    <div class="fr-col-auto fr-ml-3v">
+        <a
+            class="fr-btn fr-btn--sm fr-btn--secondary fr-btn--secondary-warning-425 fr-icon-pencil-line fr-btn--icon-left"
+            href="{{ url_for('admin.index', path='dataset/{id}/'.format(id=dataset.id)) }}"
+        >
+            {{ _('Edit') }}
+        </a>
+    </div>
+{% endif %}
+{% endcache %}
+{% endblock %}
+
 {% block content %}
 {% cache cache_duration, 'dataset-content', dataset.id|string, reuses.page|string, g.lang_code, current_user.slug or 'anonymous', dataset.last_modified|string %}
 
-{{ breadcrumb(self,
-    breadcrum_class=breadcrum_class,
-    toolbar_class=toolbar_class
-) }}
+{{ breadcrumb(self) }}
 
-<section
-    class="content fr-container fr-mb-6w">
-    {% if dataset.private %}
```

Comment:
I think we're missing the private, archive and deleted badge?

Reply:
Yes ! We talk about it with @agarrone but didn't choose where to put them